src/scripts/DropdownMenu.js

```diff
@@ -109,7 +109,7 @@ export default class DropdownMenu extends Component {
   }
 
   onKeyDown(e) {
-    if (e.keyCode === 27) { // ESC
+    if (e.keyCode === 27 || e.keyCode === 9) { // ESC || TAB
```

Comment:
this can cause other code to be broken, pass onKeyDown via props and call it here
`if(this.props.onKeyDown) this.props.onKeyDown(e)` to add custom behaviour locally